backend/src/config/app.config.spec.ts

```diff
@@ -9,9 +9,9 @@ import appConfig from './app.config';
 import { Loglevel } from './loglevel.enum';
 
 describe('appConfig', () => {
-  const domain = 'https://example.com';
-  const invalidDomain = 'localhost';
-  const rendererBaseUrl = 'https://render.example.com';
+  const baseUrl = 'https://example.com/';
```

Comment:
Instead of requiring the trailing slash everywhere, what about adding it automatically where it's needed if it is absent?

Reply:
We had that discussion before and my argument is still the same: I don't want the software to assume settings.

Reply:
The base URL can also contain a subdirectory.
If you don't require the slash then the software doesn't know if `https://example.org/asd` means "the admin forgot to remove the file and means `https://example.org/` or "the admin forgot to add the slash and means `https://example.org/asd/`.

Reply:
Okay. During chat we came to the compromise that we won't enforce the trailing slash if the user is giving only a protocol and domain. But trailing slashes are mandatory if they specify a subpath.